pages/api/resources/jobExplorer/getJobExplorerProjectData.ts

```diff
@@ -0,0 +1,20 @@
+import { devOpsEngineerProjectData } from "./devOpsEngineer/projectData";
+
+export type JobExplorerProjectData = {
+  title: string;
+  goal: string;
+  objectives: string[];
+  requirements: string[];
+  estimatedTime: string;
+  submissionRequirements: string;
+};
+
+const projectsDataMap: { [slug: string]: JobExplorerProjectData } = {
+  devOpsEngineerProject: devOpsEngineerProjectData,
+};
+
+export function getJobExplorerProjectData(
+  slug: string
+): JobExplorerProjectData | undefined {
+  return projectsDataMap[slug];
```

Comment:
this file takes the slug that was passed in and returns the project data for that slug. going forward we just need to add more key/value pairs to projectsDataMap and this function will return the right data.

Reply:
Cool, thanks for building this method; I think it will be helpful.